### PR TITLE
Release v7.1.0

### PR DESCRIPTION
**Pull request by jitu5**
## Description

# Release 7.1.0 

## Major features and improvements

- Add build command to the CLI using `kedro viz build` for creating build directory of local Kedro Viz instance with Kedro project data. (#1697)

## Bug fixes and other changes
- Set Kedro Viz start method to spawn process while launching it from Jupyter Notebook. (#1696)
- Fix bug on Plotly graph rendering. (#1701)

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: src/components/update-reminder/update-reminder-content.js [hunk 1]
```diff
@@ -1,21 +1,13 @@
 export const updateContent = {
-  date: '18 December 2023',
+  date: '22 January 2024',
   features: [
     {
-      title: 'Update CLI command `kedro viz run` which starts Kedro-viz',
+      title: 'Build Kedro-viz using cli `kedro viz build`',
```
Comment (rashidakanchwala) [inline src/components/update-reminder/update-reminder-content.js:5]:
```suggestion
      title: 'Publish Kedro-viz on any hosting platform using cli `kedro viz build`',
```

File: RELEASE.md [hunk 2]
```diff
@@ -6,14 +6,15 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-# Upcoming Release
+# Release 7.1.0
 
 ## Major features and improvements
-- Add build command to the CLI using `kedro viz build` for creating build directory of local Kedro Viz instance with static data. (#1615)
+- Add build command to the CLI using `kedro viz build` for creating build directory of local Kedro Viz instance with Kedro project data. (#1697)
```
Comment (merelcht) [inline RELEASE.md:12]:
```suggestion
- Add build command `kedro viz build` to the CLI to allow users to create a build directory of local Kedro Viz instance with Kedro project data. (#1697)
```

**Review by merelcht — APPROVED** [review]
Minor suggestion for the wording of the release notes, but otherwise LGTM! 👍

**Review by rashidakanchwala — APPROVED** [review]
Thanks @jitu5

**Review by astrojuanlu — APPROVED** [review]
🚀

**Review by NeroOkwa — APPROVED** [review]
Looks good!!!

**Pull request merged**